installer/scripts/onboard_agent.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+#
+# Easy download/install/onboard script for the OMSAgent for Linux
+#
+
+
+# Values to be updated upon each new release
+GITHUB_RELEASE="https://github.com/Microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent-201610-v1.2.0-148/"
+BUNDLE_X64="omsagent-1.2.0-148.universal.x64.sh"
+BUNDLE_X86="omsagent-1.2.0-148.universal.x86.sh"
+
+usage()
+{
+    echo "usage: $1 [OPTIONS]"
+    echo "Options:"
+    echo "  -w id, --id id             Use workspace ID <id> for automatic onboarding."
+    echo "  -s key, --shared key       Use <key> as the shared key for automatic onboarding."
+    echo "  -d dmn, --domain dmn       Use <dmn> as the OMS domain for onboarding. Optional."
+    echo "                             default: opinsights.azure.com"
+    echo "                             ex: opinsights.azure.us (for FairFax)"
+    echo "  -? | -h | --help           shows this usage text."
+}
+
+
+# Extract parameters
+while [ $# -ne 0 ]
+do
+    case "$1" in
+        -d|--domain)
+            topLevelDomain=$2
+            shift 2
+            ;;
+
+        -s|--shared)
+            onboardKey=$2
+            shift 2
+            ;;
+
+        -w|--id)
+            onboardID=$2
+            shift 2
+            ;;
+
+        -\? | -h | --help)
+            usage `basename $0` >&2
+            exit 0
+            ;;
+
+         *)
+            echo "Unknown argument: '$1'" >&2
+            echo "Use -h or --help for usage" >&2
+            exit 1
+            ;;
+    esac
+done
+
+
+# Parameter checks
+if [ -z "$onboardID" -o -z "$onboardKey" ]; then
+    echo "Must specify both workspace ID (--id) and key (--shared) to onboard" 1>& 2
+    exit 1
+fi
+
+bundleParameters="--upgrade -w $onboardID -s $onboardKey"
```

Comment:
Robbie has recommended upgrade rather than install for all scenarios. The current copyable commands in the portal are upgrade.